site/scripts/library.js

```diff
@@ -16,9 +16,19 @@ const re = RegExp('^https?://');
 export { filterStatus, filterTracking, filterSource, sortOrder, activeTabId };
 
 // Function to Initialise the Tab Contents and Library from the JSON found in the data variable.
-export function initializeLibrary() {
-  const categories = window.data.backupCategories || [];
-  let mangaItems = window.data.backupManga;
+export function initializeLibrary(fork = "mihon") {
+  let categories, mangaItems;
+  if (fork == "ani") {
+ categories = window.data.backupAnimeCategory || [];
+   mangaItems = window.data.backupAnime;
+  } else {
+  categories = window.data.backupCategories || [];
+  mangaItems = window.data.backupManga;
+  }
+  
+  window.data.backupManga = mangaItems;
+  window.data.backupCategories = categories;
+
```

Comment:
Probably easier to rename the fields in the schema and avoid this extra logic

Reply:
I updated the code